### PR TITLE
Breaking naming changes

### PR DESCRIPTION
**Pull request by MathisGD**

### CODE REVIEW

**Review by Jean-Grimal — COMMENTED** [review]
CI is failing

File: src/ERC20DelegatesUpgradeable.sol [hunk 1]
```diff
@@ -68,38 +68,40 @@ abstract contract ERC20DelegatesUpgradeable is
...
         ERC20DelegatesStorage storage $ = _getERC20DelegatesStorage();
-        address oldDelegate = delegates(delegator);
-        $._delegatee[delegator] = delegatee;
+        address newDelegatee = delegatee(delegator);
+        $._delegatee[delegator] = newDelegatee;
 
-        emit DelegateChanged(delegator, oldDelegate, delegatee);
-        _moveDelegateVotes(oldDelegate, delegatee, _getVotingUnits(delegator));
+        emit DelegateChanged(delegator, newDelegatee, newDelegatee);
+        _moveDelegateVotes(newDelegatee, newDelegatee, _getVotingUnits(delegator));
```
Comment (QGarchery) [inline src/ERC20DelegatesUpgradeable.sol:104]:
```suggestion
        emit DelegateChanged(delegator, oldDelegatee, newDelegatee);
        _moveDelegateVotes(oldDelegatee, newDelegatee, _getVotingUnits(delegator));
```

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]
I don't think this is a good idea tbh and I would keep the current naming

**Review by Jean-Grimal — APPROVED** [review]
it makes sense

File: src/ERC20DelegatesUpgradeable.sol [hunk 2]
```diff
@@ -49,7 +49,7 @@ abstract contract ERC20DelegatesUpgradeable is
     /* GETTERS */
 
     /// @dev Returns the delegate that `delegator` has chosen.
```
Comment (QGarchery) [inline src/ERC20DelegatesUpgradeable.sol:51]:
```suggestion
    /// @dev Returns the delegatee that `delegator` has chosen.
```

**Review by colin-morpho — DISMISSED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Review by Jean-Grimal — DISMISSED** [review]

**Review by MerlinEgalite — DISMISSED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Review by Jean-Grimal — APPROVED** [review]

**Pull request merged**